receiver/awslambdareceiver/README.md

```diff
@@ -36,34 +32,68 @@ The `awslambdareceiver` operates as follows:
+Table below summarizes supported signals and their sources:
+
+| Signal  | Sources                          |
+|---------|----------------------------------|
+| Logs    | S3, CloudWatch Logs subscription |
+| Metrics | S3                               |
+
+Sections below summarize how each event source is handled.
+
+### S3 Event handling
+
+S3 events are handled in the following manner:
 
-## Supported Event Sources
+- Received S3 event notification (for example, using Lambda trigger on `s3:ObjectCreated:*`)
+- Download S3 object payload
+- Decode payload using the configured encoding extension
+  - Default encoding: Preserve S3 object content as-is
+  - Custom encoding: Use specified encoding extension (for example, `awslogs_encoding` for AWS log formats)
+  - Metrics use `awscloudwatchmetricstreams_encoding` extension by default
 
-### S3 Event Notifications (`s3:ObjectCreated:*`)
-- Fetches objects from S3
-- Decodes using specified encoding extension
+### CloudWatch Logs subscription
 
-### CloudWatch Logs Subscription Filters
-- Decodes CloudWatch Logs data
-- Extracts log events
-- Uses default `awslogs_encoding` extension with cloudwatch format
+CloudWatch Logs events are handled in the following manner:
 
-## Configuration
+- Receive CloudWatch Logs subscription filter event
+- Parse the CloudWatch Logs message (note - unlike S3 events, the payload is included in the event)
+- Decode payload using the configured encoding extension
+  - Default encoding: Parse CloudWatch Logs messages to OpenTelemetry log records
+  - Custom encoding: Use specified encoding extension (for example, `awslogs_encoding` for AWS log formats)
+
+### Configurations
 
 The following receiver configuration parameters are supported.
 
-| Name                  | Description                                                                                                                                | Default     | Required |
-|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------------|-------------|----------|
-| `s3_encoding`         | Name of the encoding extension to use for S3 objects                                                                                       | "awslogs_encoding" | Optional |
+| Name                  | Description                                             |
+|:----------------------|:--------------------------------------------------------|
+| `s3:encoding`         | Optional encoder to use for S3 event processing         | 
+| `cloudwatch:encoding` | Optional encoder to use for CloudWatch event processing | 
+
+
+Consider following notes on default behaviors:
+
+- When `s3:encoding` is not specified, the receiver defaults to preserving the S3 object content as-is for logs.
+  - However, receiver attempts to detect `strings` vs `non-strings` where `Body` will be set to `string` or `byte ` accordingly.
```

Comment:
```suggestion
  - The log record's `Body` field will be a string type where the S3 object content is valid UTF-8, and otherwise will be a byte array.
```

"Attempts to detect" leaves some ambiguity, I think in this case we can be very explicit about how data is mapped since the logic is very simple.